### PR TITLE
functions: update ocr sample to use environment variables

### PR DESCRIPTION
**Pull request by codyoss**

### CODE REVIEW

File: functions/ocr/app/setup.go [hunk 1]
```diff
@@ -67,22 +59,20 @@ var (
...
-			return fmt.Errorf("os.Open: %v", err)
-		}
-
-		d := json.NewDecoder(cfgFile)
-		config = &configuration{}
-		if err = d.Decode(config); err != nil {
-			return fmt.Errorf("Decode: %v", err)
-		}
-	}
+	projectID = os.Getenv("PROJECT_ID")
```
Comment (tbpg) [inline functions/ocr/app/setup.go:71]: Should this be `GOOGLE_CLOUD_PROJECT`?

File: functions/ocr/app/ocr_test.go [hunk 2]
```diff
@@ -38,28 +38,21 @@ const (
...
-// TestMain sets up the config rather than using the config file
-// which contains placeholder values.
 func setupTests(t *testing.T) {
 	ctx := context.Background()
 	projectID = os.Getenv("GOLANG_SAMPLES_PROJECT_ID")
 	if projectID == "" {
 		t.Skip("GOLANG_SAMPLES_PROJECT_ID is unset")
 	}
-	bucketName = fmt.Sprintf("%s-result", projectID)
+	resultBucket = fmt.Sprintf("%s-result", projectID)
```
Comment (tbpg) [inline functions/ocr/app/ocr_test.go:51]: Confirming you tried deploying and running this?

File: functions/ocr/app/setup.go (hunk 1 above)
Comment (codyoss) [reply functions/ocr/app/setup.go:71]: You are right, yes.

File: functions/ocr/app/ocr_test.go (hunk 2 above)
Comment (codyoss) [reply functions/ocr/app/ocr_test.go:50]: I had, but did not realized it skiped before locally. Tested, working now.

**Pull request merged**